redis/cache/redispipebp/monitored.go

```diff
@@ -60,19 +61,40 @@ func (s MonitoredSync) Do(ctx context.Context, cmd string, args ...interface{})
 		s.Name+".do",
 		tracing.SpanTypeOption{Type: tracing.SpanTypeClient},
 	)
+	command := extractCommand(cmd)
+	active := redisprom.ActiveRequests.With(prometheus.Labels{
+		// We don't have LblClientName, LblDatabase, LblType, and LblDeployment
+		redisprom.LblPool:    s.Name,
```

Comment:
and we probably should just spell it fully as `LabelPool`. saving 2 characters really doesn't seem worth it.

Reply:
updated